postgres/parser/parser/sql.y

```diff
@@ -3434,6 +3522,58 @@ option_clause:
   {
     $$.val = tree.RoutineOption{OptionType: tree.OptionSupport, Support: $2}
   }
+
+alter_procedure_option_list:
+  alter_procedure_option
+  {
+    $$.val = []tree.RoutineOption{$1.routineOption()}
+  }
+| alter_procedure_option_list alter_procedure_option
+  {
+    $$.val = append($1.routineOptions(), $2.routineOption()) /* TODO: check for duplicate definition - "ERROR:  conflicting or redundant options" */
```

Comment:
My thought would be that this is something we'd check after the parsing step, but I haven't looked into how PostgreSQL would handle it either.

Reply:
Ah yes, I'll add handling the check after parsing for the statements that are affected by this in separate PR.